engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkConfig.scala

```diff
@@ -14,4 +14,8 @@ import scala.concurrent.duration.DurationInt
 case class FlinkConfig(restUrl: String,
                        jobManagerTimeout: FiniteDuration = 1 minute,
                        shouldVerifyBeforeDeploy: Boolean = true,
-                       shouldCheckAvailableSlots: Boolean = true)
+                       shouldCheckAvailableSlots: Boolean = true,
+                       waitForDuringDeployFinish: Option[FlinkWaitForDuringDeployFinishedConfig] =
+                       Some(FlinkWaitForDuringDeployFinishedConfig(60, 1 second)))
```

Comment:
Now to turn it off in config, one has to write:
```
waitForDuringDeployFinish {} 
```
?
It looks a bit weird.
Maybe it's better to introduce `waitForDuringDeployFinish.enabled` flag or get rid of `Option` ? So this checking behaviour is required.

Reply:
You're right - it can be inconvenient. I added enabled option